components/ui/Footer.js

```diff
@@ -1,15 +1,51 @@
-"use client"
+"use client";
 
-import { usePathname } from 'next/navigation';
-import styles from './Footer.module.css';
+import { usePathname } from "next/navigation";
+import styles from "./Footer.module.css";
+import { Children } from "react";
+
+const SocialMediaList = [
```

Comment:
Small thing:
Only the React components or classes (or Typescript types) should have a starting capital letter. 
This is a simple JS variable so:
`socialMediaList`